Packs/ctf01/Integrations/OHMYVTCTF/OHMYVTCTF.py

```diff
@@ -0,0 +1,135 @@
+        'pe_resource_parents': EntityRelationship.Relationships.EXECUTED_BY,
+        'similar_files': EntityRelationship.Relationships.SIMILAR_TO,
+    },
+    'domain': {
+        'cname_records': EntityRelationship.Relationships.IS_ALSO,
+        'caa_records': EntityRelationship.Relationships.RELATED_TO,
+        'communicating_files': EntityRelationship.Relationships.DROPS,
+        'downloaded_files': EntityRelationship.Relationships.DROPS,
+        'immediate_parent': EntityRelationship.Relationships.SUB_DOMAIN_OF,
+        'mx_records': EntityRelationship.Relationships.RELATED_TO,
+        'ns_records': EntityRelationship.Relationships.DROPS,
+        'parent': EntityRelationship.Relationships.SUB_DOMAIN_OF,
+        'referrer_files': EntityRelationship.Relationships.RELATED_TO,
+        'resolutions': EntityRelationship.Relationships.RESOLVED_FROM,
+        'siblings': EntityRelationship.Relationships.SUPRA_DOMAIN_OF,
+        'soa_records': EntityRelationship.Relationships.IS_ALSO,
+        'subdomains': EntityRelationship.Relationships.SUPRA_DOMAIN_OF,
+        'urls': EntityRelationship.Relationships.HOSTS,
+    }, 'ip': {
+        'communicating_files': EntityRelationship.Relationships.COMMUNICATES_WITH,
+        'downloaded_files': EntityRelationship.Relationships.DROPS,
+        'referrer_files': EntityRelationship.Relationships.RELATED_TO,
+        'resolutions': EntityRelationship.Relationships.RESOLVES_TO,
+        'urls': EntityRelationship.Relationships.RELATED_TO,
+    }, 'url': {
+        'contacted_domains': EntityRelationship.Relationships.RELATED_TO,
+        'contacted_ips': EntityRelationship.Relationships.RELATED_TO,
+        'downloaded_files': EntityRelationship.Relationships.DROPS,
+        'last_serving_ip_address': EntityRelationship.Relationships.RESOLVED_FROM,
+        'network_location': EntityRelationship.Relationships.RESOLVED_FROM,
+        'redirecting_urls': EntityRelationship.Relationships.DUPLICATE_OF,
+        'redirects_to': EntityRelationship.Relationships.DUPLICATE_OF,
+        'referrer_files': EntityRelationship.Relationships.EMBEDDED_IN,
+        'referrer_urls': EntityRelationship.Relationships.RELATED_TO,
+    }
+}
+
+
+'''
+            __,__
+   .--.  .-"     "-.  .--.
+  / .. \/  .-. .-.  \/ .. \
+ | |  '|  /   Y   \  |'  | |
+ | \   \  \ 0 | 0 /  /   / |
+  \ '- ,\.-"`` ``"-./, -' /
+   `'-' /_   ^ ^   _\ '-'`
+       |  \._   _./  |
+       \   \ `~` /   /
+        '._ '-=-' _.'
+           '~---~'
+!!!Opppppssssss!!!
+Someone messed with the integarion  maybe its related to something?
+
+# Flag: <Which animal is in the picture?>
+'''
+
+
+def test_module():
+    """
+    Returning 'ok' indicates that the integration works like it suppose to. Connection to the service is successful.
+
+    Args:
+        client: HelloWorld client
+
+    Returns:
+        'ok' if test passed, anything else will fail the test
+    """
+    return 'ok'
+
+
+def main():
```

Comment:
What this integration is used for? completely empty

Reply:
this integration is just a placeholder, we hide a clue for the answer inside the py/yml file. however, we can add a simple print command or something like that if that bothers you.